Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py

```diff
@@ -155,7 +155,7 @@ def parse_response(resp):
             return resp
         try:
             return resp.json()
-        except Exception, ex:
+        except Exception as ex:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.